src/components/settings/ProvisioningSettings.vue

```diff
@@ -243,7 +243,9 @@
 						/>
 						<br />
 						<small>{{
-							t('mail', "* %USERID% and %EMAIL% will be replaced with the user's UID and email")
+							t('mail', "* %USERID% %DISPLAYNAME and %EMAIL% will be replaced with the user's UID, DISPLAYNAME and email")
```

Comment:
please remove the two lines above. admins should know the values of the three placeholders and thus be able to figure out what will work for them :)

Reply:
is it better now? But please think of not so experienced admins, the documentation of the mail app is really sparse

Reply:
It should be "* %USERID%, %DISPLAYNAME% and %EMAIL% will be replaced with the user's UID, email and display name."

Hm. I'm fine with adding more docs. But they should be written in a form that is comprehensible for everyone and not a listing of edge case recommendations.

Reply:
LDAP isn't an edge case in large environments 

Greetings from CS3

Reply:
could you tell me where the error in eslint comes from?  I was not yet able to setup the complete build environment locally

Reply:
> Greetings from CS3

Say hi to Björn and Frank :)

> could you tell me where the error in eslint comes from? I was not yet able to setup the complete build environment locally

ESLint want the `t` call to be wrapped into multiple lines. You can run `npm run lint:autofix` to resolve this automagically.

Reply:
Arghh, such a picky thing
i got it now (blind test, as I didn't get eslint running on my Ubuntu LTS 18.04)
but now xml-linters failed, looks like it had connection trouble to apps.nextcloud.com